Signal/src/call/CallService.swift

```diff
@@ -1152,6 +1161,33 @@ protocol CallServiceObserver: class {
         // Do nothing
     }
 
+    internal func callRecordDidChange(call: SignalCall, callRecord: TSCall?) {
+        AssertIsOnMainThread()
+
+        updateCallRecordType(call: call)
+    }
+
+    internal func updateCallRecordType(call: SignalCall?) {
+        AssertIsOnMainThread()
+
+        guard call != nil else {
```

Comment:
nit: ditto about rebinding `call` and `callRecord` to a local non-optional.

```
guard let call = self.call else {
    return
}
```

then use the non-optional `call` (sans `!`) in the rest of the method.